libse/SubtitleFormats/PhoenixSubtitle.cs

```diff
@@ -67,10 +77,8 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                             line = line.Trim(TrimChars);
                         }
 
-                        paragraph.Number = i + 1;
```

Comment:
Don't assume all the lines will be parsed correctly (we now allow ignoring white-line), so the `i` is not a good candidate here. 😞